new_platforms/vscode-extension/assets/baseSolutionTemplateFolder/[[project-name]].edl

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+enclave {
+    from "openenclave/stdio.edl" import *;
```

Comment:
It would be good to align this EDL file with the VS Extension's EDL file.   Are the differences intentional?

Reply:
yes.  the vs extension edl doesn't allow me to actually show that the enclave is doing anything.  this edl has the same problem.

my preference is to change the edl to allow the enclave to do something (probably some string manipulation) that can be visualized in the iot edge scenario.

Reply:
i've used ecall_ and ocall_ prefixes as the vs extension does ... but otherwise, the edl remains different.